dsa-23au/java-dsa/pswish-natmcl/pswish-app/src/main/java/com/pswishcorp/app/Time_Driving_Spreadsheet.csv

```diff
@@ -0,0 +1,27 @@
+﻿Time,Date,Elapsed,Orig,Dest,Detour enroute,,Driver,Distance
+830,9/26/2023,28 min,Lakewood,Evergreen,no,,Nathan,21 miles
+1540,9/26/2023,32 min,Evergreen,Lakewood,no,,Nathan,21 miles
+830,9/28/2023,26 min,Lakewood,Evergreen,no,,Nathan,21 miles
+1505,9/28/2023,66 min,Evergreen,Lakewood,no,,Nathan,21 miles
+830,9/29/2023,40 min ,Lakewood,Evergreen,no,,Nathan,21 miles
+1807,9/29/2023,30 min,Evergreen,Lakewood,no,,Nathan,21 miles
+905,10/2/2023,32 min,Lakewood,Evergreen,no,,Nathan,21 miles
+1908,10/2/2023,35 min,Evergreen,Lakewood,no,,Nathan,21 miles
+840,10/3/2023,28 min,Lakewood,Evergreen,no,,Nathan,21 miles
+1510,10/3/2023,28 min,Evergreen,Lakewood,no,,Nathan,21 miles
+905,10/5/2023,30 min,Lakewood,Evergreen,no,,Nathan,21 miles
+1508,10/5/2023,28 min,Evergreen,Lakewood,no,,Nathan,21 miles
+845,10/9/2023,27 min,Lakewood,Evergreen,no,,Nathan,21 miles
+1909,10/9/2023,30 min,Evergreen,Lakewood,no,,Nathan,21 miles
+835,10/10/2023,29 min,Lakewood,Evergreen,no,,Nathan,21 miles
+1510,10/10/2023,27 min,Evergreen,Lakewood,no,,Nathan,21 miles
+850,10/12/2023,28 min,Lakewood,Evergreen,no,,Nathan,21 miles
+1510,10/12/2023,33 min,Evergreen,Lakewood,no,,Nathan,21 miles
+1505,10/13/2023,48 min,Lakewood,Evergreen,no,,Nathan,21 miles
+1810,10/13/2023,84 min,Evergreen,Lakewood,no,,Nathan,21 miles
+830,10/16/2023,30 min,Lakewood,Evergreen,no,,Nathan,21 miles
+1905,10/16/2023,35 min,Evergreen,Lakewood,no,,Nathan,21 miles
+840,10/17/2023,28 min,Lakewood,Evergreen,no,,Nathan,21 miles
+1507,10/17/2023,29 min,Evergreen,Lakewood,no,,Nathan,21 miles
+,10/19/2023,,Lakewood,Evergreen,no,,Nathan,21 miles

```

Comment:
This looks to be missing the time entry, also, between "no" and "Nathan" you have an extra `,`   is this intentional like a placeholder?